docs/src/pages/landing-page/Index.vue

```diff
@@ -196,4 +304,22 @@ export default defineComponent({
 .support {
   height: 100vh;
 }
+
+.my-grid {
+  font-size: .5rem;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-column-gap: 1rem;
+  grid-row-gap: 2rem;
+  align-content: center;
+}
+
+.final-words-section {
+  // TODO: reduce image height (from top), and remove this negative margin
+  margin-top: -30%;
+}
+
+.showcase-cards {
+  overflow: hidden;
```

Comment:
This doesn't seem to do much, what is it meant for?

Reply:
This is because of the responsiveness of the heights of the twitter cards, if for some reason a card (maybe the content of the tweet changed or so?) was larger than the height of the carousel specified then it will overflow and create horizontal scrollbars. Overflow hidden will prevent a weird the weird looking scrollbars.

Reply:
Strange, could not reproduce when I was testing it
Anyway, in these cases **always** add an explanation about why that's needed
Remember your code should always be understandable by a stranger (or yourself in 6 months, which is basically the same), add comments accordingly every time you add need to fix a strange interaction which isn't obvious